packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -15,6 +15,7 @@ const WebpackOnBuildPlugin = require('on-build-webpack');
 const tar = require('tar');
 const glob = require('glob');
 const execSync = require('child_process').execSync;
+const allPossibleCategories = require('@joplin/lib/plugin-categories.json').allPossibleCategories;
```

Comment:
`require` statements should be at the top of the file

Reply:
Do you mean this `allPossibleCategories` statement should be on top of other `require` statements? Because it is already on top of the `webpack.config.js`.

Reply:
right, sorry that's fine as it is then